docs/reference/release-notes.md

```diff
@@ -0,0 +1,33 @@
+# COS 2 release notes
+October 2025
+
+These release notes cover new features and changes in COS 2.
+
+COS 2 is designated as a short-term support release. This means it will continue to receive security updates and critical bug fixes for 9 months.
+
+See our [release policy](release-policy.md) and [upgrade instructions](../how-to/upgrade.md).
+
+## Requirements and compatibility
+See [system requirements](system-requirements.md).
```

Comment:
If these are general requirements that shouldn't be changing on a per-release basis, then you can keep it as is with the template.

If these change depending on the release, I think it's fine to deviate from the template and put that information directly in the release notes.

You can decide what's best for your product - just wanted to add that perspective since this is a newer process for your team :)

Reply:
Thanks!
Generally sysreqs shouldn't change from release to release, but even if they do, we'd have it covered anyway because the docs are versioned. Unless I'm missing something?

Reply:
Ahh, right. Yea that's fine then, I forgot your docs are versioned anyway